py/desispec/scripts/submit_night.py

```diff
@@ -143,16 +156,17 @@ def submit_night(night, proc_obstypes=None, z_submit_types=None, queue='realtime
 
     ## Load in the files defined above
     etable, ptable = load_tables(tablenames=table_pathnames, tabletypes=table_types)
+    full_etable = etable.copy()
 
     ## filter by TILEID if requested
     if tiles is not None:
         log.info(f'Filtering by tiles={tiles}')
         if etable is not None:
             keep = np.isin(etable['TILEID'], tiles)
             etable = etable[keep]
-        if ptable is not None:
-            keep = np.isin(ptable['TILEID'], tiles)
-            ptable = ptable[keep]
+        #if ptable is not None:
+        #    keep = np.isin(ptable['TILEID'], tiles)
+        #    ptable = ptable[keep]
```

Comment:
Please confirm that this change (and similar one below) is purposeful and not a leftover from testing.  i.e. we used to filter ptable on tiles, now we don't filter on tiles, and that surprises me, but perhaps the equivalent functionality is happening somewhere else in the chain?

Reply:
The survey and tile code was added by you. I don't see why we would filter previously submitted jobs based on input criteria for future jobs. It throws away information and I didn't understand what the benefit was. Other than being a record of what was done the only other thing the processing tables are used for is as a list of things that have already been processed (so that we don't re-process them again every time the code is run).

Do you happen to remember what your motivation was for doing that? Is there something that I am missing?

Reply:
OK, it sounds like this might have been a misunderstanding on my part about what needed to be filtered to form the reference set of what needed to be processed.  I verified that
```
desi_run_night -n 20210408 --tiles 191 --dry-run
```
does what is expected with only processing exposures related to tile 191, even if other tiles previously existed in the processing table.